### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Update README

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,9 @@ If you encounter abusive behavior violating the [Ansible Code of Conduct](https:
 
 ## Contributing to this collection
 
-The content of this collection is made by [people](https://github.com/ansible-collections/community.postgresql/blob/main/CONTRIBUTORS) like you, a community of individuals collaborating on making the world better through developing automation software.
+The content of this collection is made by [people](https://github.com/ansible-collections/community.postgresql/blob/main/CONTRIBUTORS) just like you, a community of individuals collaborating on making the world better through developing automation software.
+
+We are activelly accepting new contributors.
```
Comment (Andersson007) [inline README.md:18]:
```suggestion
We are actively accepting new contributors.
```

**Comment by Andersson007** [comment]
ready for review

**Review by tcraxs — APPROVED** [review]
LGTM

**Comment by Andersson007** [comment]
@tcraxs thanks for reviewing!
(Would you like to discuss commit here? If yes, please contact me via email, thanks!)

**Pull request merged**